src/components/PostPageDetail/elements/Content.js

```diff
@@ -184,16 +185,17 @@ const Content = ({
                   {hashtagAtComponent(message, null, isShortText())}
                 </Text>
               ) : (
-                <Text
-                  style={[
+                <MemoLinkDetectionText
+                  text={message}
+                  withTopicDetection={true}
+                  parentTextStyle={[
                     styles.textContentFeed(isShortText(), parentData?.color),
                     {
                       fontSize: font,
                       lineHeight
                     }
-                  ]}>
-                  {hashtagAtComponent(sanitizeUrl(message), null, isShortText())}{' '}
-                </Text>
+                  ]}
+                />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the `<Text>` component with the `<MemoLinkDetectionText>` component is a good change. This will provide better handling for texts that contain links, improving the user experience.
```diff
-                <Text
-                  style={[
-                     styles.textContentFeed(isShortText(), parentData?.color),
-                     {
-                       fontSize: font,
-                       lineHeight
-                     }
-                  ]}>
-                  {hashtagAtComponent(sanitizeUrl(message), null, isShortText())}{' '}
-                </Text>
+                <MemoLinkDetectionText
+                  text={message}
+                  withTopicDetection={true}
+                  parentTextStyle={[
+                     styles.textContentFeed(isShortText(), parentData?.color),
+                     {
+                       fontSize: font,
+                       lineHeight
+                     }
+                  ]}
+                />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->